server/monitor.go

```diff
@@ -238,14 +268,42 @@ func (s *Server) HandleConnz(w http.ResponseWriter, r *http.Request) {
 		}
 
 		// Fill in user if auth requested.
-		if auth == 1 {
+		if auth {
 			ci.AuthorizedUser = client.opts.Username
 		}
 
 		client.mu.Unlock()
 		i++
 	}
+	return c, nil
+}
+
+// HandleConnz process HTTP requests for connection information.
+func (s *Server) HandleConnz(w http.ResponseWriter, r *http.Request) {
+	sortOpt := SortOpt(r.URL.Query().Get("sort"))
+	auth, _ := strconv.Atoi(r.URL.Query().Get("auth"))
```

Comment:
Capture errors I think? Can I crash server with simple bad cgi args?

Reply:
I did not change that. The Atoi may return an error but `auth` or `subs` will then default to 0. As of now, the only possible values were `1`, so anything else will default to 0 (or `false` for the options).

Reply:
As as said, it won't crash the server. An invalid sort option for instance will be rejected in Connz(), so HandleConnz() will return an error page. For auth and others, we default to default values (for instance if offset is < 0 we use 0, etc..). Should I still capture and report invalid values for those here?

Reply:
I think we should know, longer term could be a form of attack.

Reply:
Ok, done.

Reply:
LGTM